CONTRIBUTING.md

```diff
@@ -158,3 +223,319 @@ including how to setup a local Kubernetes cluster, can be found in
+
+### Issue Workflow
+
+The purpose of this workflow is to formalize a lightweight set of processes that
+will optimize issue triage and management which will lead to better release
+predictability and community responsiveness for support and feature
+enhancements. Additionally, Antrea must prioritize issues to ensure interlock
+alignment and compatibility with other projects including Kubernetes. The
+processes described here will aid in accomplishing these goals.
+
+![developer workflow overview](docs/assets/developer-workflow-opaque-bg.png)
+
+For additional details and to improve the metadata associated with your issues,
+please keep reading.
+
+### New Issues
+
+When submitting a new issue, please search the existing set of issues
+to see if an issue has already been created that addresses your concern.
+
+When creating your new issue, please ensure that you:
+
+* label the issue with the appropriate [`kind/<?>`](#kinds) label.  The kind of
+  issue will determine the issue workflow.
+* label the issue with the appropriate [`area/<?>`](#area-assigment) label. The
+  area label will identify the area of interest an issue or PR belongs in and
+  will ensure the appropriate reviewers shepherd the issue or PR through to its
+  closure.
+* for issues, label the estimated size (if known). The size can be updated
+  during backlog grooming by the contributors. This estimate is used to guide
+  sprint scoping the exercise.
+
+### Issue Triage
+
+Once an issue has been submitted, the CI (GitHub actions) or a human will
+automatically review the submitted issue or PR to ensure that it has all relevant
+information. If information is lacking or there is another problem with the
+submitted issue, an appropriate [`triage/<?>`](#triage) label will be
+applied.
+
+After an issue has been triaged, the maintainers can prioritize the issue with
+an appropriate [`priority/<?>`](#priority) label.
+
+Once an issue has been submitted, categorized, triaged, and prioritized it
+is marked as `ready-to-work`. A ready-to-work issue should have labels
+indicating assigned areas, prioritization, and should not have any remaining
+triage labels.
+
+### Issue Kinds
+
+Use an kind label to describe the kind of issue you are submitting. Valid
+kinds include:
+
+* [`kind/api-change`](#api-change) -- for api changes
+* [`kind/bug`](#bug) -- for filing a bug
+* [`kind/cleanup`](#cleanup) -- for code cleanup and organization
+* [`kind/deprecation`](#deprecation) -- for deprecating a feature
+* [`kind/design`](#design) -- for proposing a design or architectural change
+* [`kind/documentation`](#documentation) -- for updating documentation
+* [`kind/failing-test`](#failing-test) -- for reporting a failed test (may
+  create with automation in future)
+* [`kind/feature`](#feature) -- for proposing a feature
+* [`kind/support`](#support) -- to request support (please try using our
+  [Slack](https://kubernetes.slack.com/archives/CR2J23M0X) channel for
```

Comment:
I suggest pointing back to the `Accounts Setup` section here instead, since if someone has not joined the K8s wroskpace already, the link will not work

Reply:
great suggestion, will change.